content/docs/start/index.md

```diff
@@ -69,7 +69,7 @@ in two independent trails:
   be attached to pipelines. These let you capture, navigate, and evaluate ML
   projects without leaving Git. Think "Git for machine learning".
 
-### Experiments Trail
+### Experimentation
 
 - [**Experiments**](/doc/start/experiments) enable exploration, iteration, and
   comparison across many ML experiments. Track your experiments with automatic
```

Comment:
Should we call this 'tail' something longer? Maybe "Machine Learning Experimentation" ?